lib/vagrant-registration/action/register.rb

```diff
@@ -42,7 +37,8 @@ def call(env)
 
           @logger.debug('Registration is skipped due to the configuration') if config.skip
 
-          @app.call(env) if Plugin.vbguest_plugin?
+          # Call next middleware in chain
+          @app.call(env)
```

Comment:
This cannot be done, as it does not work with libvirt provider: http://paste.fedoraproject.org/343422/58251114

However the hook and the detection of VirtualBox provider could work. I will test that tomorrow.